addons/website_sale/static/src/js/website_sale.js

```diff
@@ -8,7 +8,7 @@ var _t = core._t;
 var timeout;
 
 publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
-    selector: '#top_menu a[href$="/shop/cart"]',
+    selector: 'a.nav-link[href$="/shop/cart"]',
```

Comment:
Probably ok but I am a bit afraid of targeting `.nav-link`. The header templates are being re-made at the moment and I am sure we'll end up with a cart link not using the nav-link class at some point ^^

Maybe `'#top a[href$="/shop/cart"]'` instead of `'#top_menu a[href$="/shop/cart"]'` is ok ?